content/en/network_monitoring/devices/netflow.md

```diff
@@ -85,6 +85,13 @@ You can find the NetFlow page on the [Network Devices page][5].
 This data is also available in Dashboards, Notebooks, and more for more precise queries and for correlating with other sources of data.
 {{< img src="network_device_monitoring/netflow/notebook.png" alt="Notebook" width="80%" >}}
 
+## Sampling rate
+
+Netflow sampling rate is taken into account in the computation of bytes and packets by default. This means that the displayed values for bytes and packets are computed with the sampling rate applied.
```

Comment:
I think our convention is to capitalize the F in NetFlow.

Also, this sentence is a bit confusing - referring to displayed values for bytes and packets - when below we refer to Bytes/Packets (Adjusted) and (Sampled).

Reply:
Hum I agree with you but it is still written Bits and Packets on the NetFlow page so I am referring to them.
The Bytes/Packets (Adjusted) and (Sampled) naming is the one used on the dashboards and notebooks